src/bias_madlibs.py

```diff
@@ -105,23 +104,24 @@ def actual_label():
     else:
       return random.choice(('BAD', 'NOT_BAD'))
 
-  print('Text,Label')
+  print('Text,Label,Template')
   for template_pair in madlibber.TEMPLATE_PAIRS:
     # TODO(jetpack): here's a limit to the number of unique examples each
-    # template can produce, so bound the number of attempts at 2x the requested
-    # number. this is a hack.
+    # template can produce, so bound the number of attempts. this is a terrible
```

Comment:
It's hack. Not sure how terrible it is. :)

Reply:
Agreed :-) (Took over this change from Jetpack, terrible was his descriptor!)